internal/testdata/TestBuild_merge_YAML

```diff
@@ -128,83 +82,130 @@
-                                "amount": "17.25"
+                                "code": "RED",
+                                "percent": "10.0%"
                             }
-                        ],
-                        "base": "5172.50",
-                        "amount": "1067.25"
+                        ]
                     },
                     {
+                        "amount": "600.00",
+                        "base": "4000.00",
                         "code": "IRPF",
-                        "retained": true,
                         "rates": [
                             {
-                                "code": "STD",
+                                "amount": "600.00",
                                 "base": "4000.00",
-                                "percent": "15.0%",
-                                "amount": "600.00"
+                                "code": "STD",
+                                "percent": "15.0%"
                             }
                         ],
-                        "base": "4000.00",
-                        "amount": "600.00"
+                        "retained": true
                     }
                 ],
                 "sum": "467.25"
             },
-            "total_with_tax": "5654.75",
-            "outlays": "0.00",
-            "payable": "5654.75"
+            "total": "5187.50",
+            "total_with_tax": "5654.75"
         },
-        "payment": {
-            "terms": {
-                "code": "instant"
-            },
-            "instructions": {
-                "code": "credit_transfer",
-                "credit_transfer": [
-                    {
-                        "iban": "ES06 0128 0011 3901 0008 1391",
-                        "name": "Bankinter"
-                    }
-                ]
-            }
-        }
+        "uuid": "3d7fdbdc-d037-11eb-a068-3e7e00ce5635"
+    },
+    "head": {
+        "dig": {
+            "alg": "sha256",
+            "val": "7f90caa1a46526ab35c8c80a03fbf9cbdc97a0b6aa9d286dcb82143fe23bbb73"
+        },
+        "uuid": "9d8eafd5-77be-11ec-b485-5405db9a3e49"
     },
     "sigs": [
-        "eyJhbGciOiJFUzI1NiIsImtpZCI6ImE2YzM5MjBkLThlMzMtNDE1OS1iMzM3LTllNzQ2MTcxNmRmMSJ9.eyJ1dWlkIjoiOWQ4ZWFmZDUtNzdiZS0xMWVjLWI0ODUtNTQwNWRiOWEzZTQ5IiwidHlwIjoiYmlsbC5JbnZvaWNlIiwicmduIjoiRVMiLCJkaWciOnsiYWxnIjoic2hhMjU2IiwidmFsIjoiZGNlM2JjM2M4YmYyOGYzZDIwOWY3ODM5MTdiMzA4MmRkYzAzMzlhNjZlOWJhM2FhNjM4NDllNDM1N2RiMTQyMiJ9fQ.CbBRUWrP1ELBmsoF9KnMWB5D6f1rNkai4lMZYx83vP2oKt3XdIMv-kvME4voFWN7XDjmsFb8dnt1sD00Oi54rQ"
+        "eyJhbGciOiJFUzI1NiIsImtpZCI6ImE2YzM5MjBkLThlMzMtNDE1OS1iMzM3LTllNzQ2MTcxNmRmMSJ9.eyJ1dWlkIjoiOWQ4ZWFmZDUtNzdiZS0xMWVjLWI0ODUtNTQwNWRiOWEzZTQ5IiwidHlwIjoiYmlsbC5JbnZvaWNlIiwicmduIjoiRVMiLCJkaWciOnsiYWxnIjoic2hhMjU2IiwidmFsIjoiZGNlM2JjM2M4YmYyOGYzZDIwOWY3ODM5MTdiMzA4MmRkYzAzMzlhNjZlOWJhM2FhNjM4NDllNDM1N2RiMTQyMiJ9fQ.CbBRUWrP1ELBmsoF9KnMWB5D6f1rNkai4lMZYx83vP2oKt3XdIMv-kvME4voFWN7XDjmsFb8dnt1sD00Oi54rQ",
```

Comment:
I don't think this is correct, there shouldn't be two signatures from build. Comparing them both on jwt.io indicates the Digest values do not match. I'd look at the source of these as I do think it's important we're testing against valid data.